controller/postController.js

```diff
@@ -72,6 +73,8 @@ const addCommentOnPost = async (req, res) => {
     const postData = await postRef.get();
     if (postData.exists) {
       await postRef.collection("comments").add({ userUid, comment, createdAt: new Date(), updatedAt: new Date() })
+      const postUserUid = postData.updatedBy
```

Comment:
why updatedBy? why not createdBy? @ashok-choudhary-0

Reply:
sir in assignment it is mentioned that fieldname will be updatedBy so thats why i use updatedBy  when user creates a new post